europe/README.EUROPE

```diff
@@ -457,4 +457,26 @@ only used as steps in more complicated transformations.
 
 Init file with various transformations and conversions of relevance in Iceland.
 
-* ISL
\ No newline at end of file
+* ISL
+
+### Hungary: ETRS89 HD72/EOV (EPSG:23700)
```

Comment:
can you move that before the "## Included init-files" section ?

Reply:
There was no approache to EPSG so far.

Reply:
> There was no approache to EPSG so far.

As underlined in CONTRIBUTING.md, that would be very good to do so, otherwise PROJ has to act as a 'geodetic registry' to create custom database records that register the transformation grids. EPSG has much more experience than us to do that and in a consistent way